package.json

```diff
@@ -158,6 +158,14 @@
           "description": "Enable CodeLens for displaying the reproduction path",
           "default": true
         },
+        "codechecker.editor.customBugSeverities": {
```

Comment:
Maybe it would be more user friendly to use an enum setting:

    "codechecker.editor.customBugSeverities": {
          "type": "string",
          "default": "Error",
          "enum": [
            "Error",
            "Warning",
            "Information",
            "Hint"
          ],
          "description": "Control how a bug is displayed in the editor, depending on what its severity is. Bugs can be displayed as 'Error', 'Warning', 'Information' or 'Hint'. By default, everything except 'STYLE' severity is displayed as an error."
        }

so this setting will be a selection:

![image](https://github.com/Ericsson/CodecheckerVSCodePlugin/assets/77051417/dd03b268-5f43-410c-b28b-e4ecf9702c29)